src/org/opensolaris/opengrok/authorization/AuthorizationFramework.java

```diff
@@ -160,6 +160,12 @@ public boolean isAllowed(HttpServletRequest request, Project project) {
             public boolean decision(IAuthorizationPlugin plugin) {
                 return plugin.isAllowed(request, project);
             }
+
+            @Override
```

Comment:
The indentation is a bit confusing here, it'd be nice if the overrides were shifter to the right past the first column of the `new AuthorizationEntity` line